tests/integration-tests/src/test/java/org/apache/activemq/artemis/tests/integration/openwire/BasicOpenWireTest.java

```diff
@@ -64,6 +64,9 @@
    @Before
    public void setUp() throws Exception {
       super.setUp();
+      System.setProperty("org.apache.activemq.transport.AbstractInactivityMonitor.keepAliveTime", "5");
```

Comment:
shouldn't you remove this property on tearDown?

What if another test is expecting this property differently? even if it's not the case now, I think you should clear it on tearDown.